lib/pure/bitops.nim

```diff
@@ -479,24 +483,24 @@ when useGCC_builtins:
 
 elif useVCC_builtins:
   # Counts the number of one bits (population count) in a 16-, 32-, or 64-byte unsigned integer.
-  proc builtin_popcnt16(a2: uint16): uint16 {.
-      importc: "__popcnt16", header: "<intrin.h>", noSideEffect.}
-  proc builtin_popcnt32(a2: uint32): uint32 {.
-      importc: "__popcnt", header: "<intrin.h>", noSideEffect.}
-  proc builtin_popcnt64(a2: uint64): uint64 {.
-      importc: "__popcnt64", header: "<intrin.h>", noSideEffect.}
+  func builtin_popcnt16(a2: uint16): uint16 {.
```

Comment:
optional: turn into more readable 1-liners?
```nim
func builtin_popcnt16(a2: uint16): uint16 {.importc: "__popcnt16", header: "<intrin.h>".}
```
(even if exceeds that (IMO silly) 80 col limit a bit)

Reply:
let's defer this to an eventual followup PR instead